website/docs/docs/cloud/connect-data-platform/connect-onehouse.md

```diff
@@ -0,0 +1,85 @@
+---
+title: "Connect Onehouse"
+id: connect-onehouse
+description: "Setup instructions for connecting Onehouse to dbt Cloud"
+sidebar_label: "Connect Onehouse"
+pagination_next: null
+---
+
+<Snippet path="dbt-onehouse" />
+
+<Constant name="cloud" /> Supports connecting to [Onehouse SQL](https://www.onehouse.ai/) using the Apache Spark Connector with the Thrift method.
+
+**Connect to a Onehouse SQL Cluster with the `dbt-spark` adapter.**
+
+## Requirements
+
+* For dbt Cloud, ensure your Onehouse SQL endpoint is accessible via external DNS/IP, whitelisting dbt Cloud IPs.
+* For dbt Core, you can SSH tunnel into the VPC to access the endpoint, without opening up ports.
+
+## What works 
+
+* All dbt features like `dbt clean`, `dbt compile`, `dbt debug`, `dbt seed`, and `dbt run`.
+* dbt materialized type of "table" and "incremental" 
+* Apache Hudi table types of Merge on Read (MoR) and Copy on Write (CoW). It is recommended to use MoR for mutable workloads.
+
+## Limitations
+
+* dbt materialized type cannot be "view".
+* `dbt seed` has row / record limits.
+* `dbt seed` only supports CoW tables.
+
+## dbt connection
+
+Fill in the following fields when creating an **Apache Spark** warehouse connection using the Thrift connection method:
+
+| Field | Description | Examples |
+| ----- | ----------- | -------- |
+| Method | The method for connecting to Spark | Thrift |
+| Hostname | The hostname of your Onehouse SQL Cluster endpoint | `yourProject.sparkHost.com` |
+| Port | The port to connect to Spark on | 10000 |
+| Cluster | Onehouse does not use this field | |
+| Connection Timeout | Number of seconds after which to timeout a connection | 10 |
+| Connection Retries | Number of times to attempt connecting to cluster before failing | 0 |
+| Organization | Onehouse does not use this field | |
```

Comment:
I would just remove the fields that Onehouse does not require for the connection (especially since most of them related to Databricks).

Reply:
I put it there because the dbt cloud UI has those fields.

Reply:
I wish there were a way to make this cleaner 🤔